CHANGELOG.md

```diff
@@ -14,12 +14,15 @@ changes.
   numeric `eventId`.
 
 - Make `hydra-node` support time bounded transactions.
-  + Introduce time in the Hydra Head transactions. 
+  + Introduce time in the Hydra Head transactions.
   + Users have the option to submit transactions using time validity ranges.
   + Changes to the logs:
     - Added current chain slot in OpenState.
     - Added current chain slot and time in chain Tick.
 
+- `hydra-node --version` always displays git revision (SHA) alongside
```

Comment:
I don't care *that* much but if it's easy, I would prefer to not always give the revision (SHA) but only when it's not a released version so that we avoid giving the user too much noise.

But I'm really fine with that already.

Reply:
I tend to have the same opinion. However, we have not found a way to get anything else from the nix flake than the revision. Related issues: https://github.com/NixOS/nix/issues/7201 and https://github.com/NixOS/nix/issues/4682